docs/api.rst

```diff
@@ -11749,3 +11749,204 @@ Handling user notes
    :statuscode 400: Provided JSON is in some way malformed.
    :statuscode 409: No user is currently logged in. User note with the given title already exists. User has reached the limit of available notes. Check error message.
    :statuscode 500: Internal rotki error.
+
+
+Custom Assets
+================
+
+.. http:post:: /api/(version)/assets/custom
+
+   Doing a POST on this endpoint will return all the custom assets present in the database using the filter parameters.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      POST /api/1/assets/custom HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {"name": "land", "custom_asset_type": "real estate"}
+
+   :reqjson int[optional] limit: This signifies the limit of records to return as per the `sql spec <https://www.sqlite.org/lang_select.html#limitoffset>`__.
+   :reqjson int[optional] offset: This signifies the offset from which to start the return of records per the `sql spec <https://www.sqlite.org/lang_select.html#limitoffset>`__.
+   :reqjson string[optional] name: The name of the custom asset to be used as a filter.
+   :reqjson string[optional] identifier: The identifier of the custom asset to be used as a filter.
+   :reqjson string[optional] custom_asset_type: The type of the custom asset to be used as a filter.
+   :reqjson list[string] order_by_attributes: This is the list of attributes of the custom asset by which to order the results. By default we sort using ``name``.
```

Comment:
Shouldn't orrder_by_attributes and ascending be optional?

Reply:
yeah, they're optional. I'd add that. Thanks